lib/grammars.coffee

```diff
@@ -134,6 +134,22 @@ module.exports =
       command: "julia"
       args: (context) -> [context.filepath]
 
+  Kotlin:
+    "Selection Based":
+      command: "bash"
+      args: (context) ->
+        code = context.getCode(true)
+        tmpFile = GrammarUtils.createTempFileWithCode(code, ".kt")
+        jarName = context.filename.replace(/\.kt$/g, "") + ".jar"
```

Comment:
Good point. I had done the concat before I realized kotlinc required a `kt` extension.